src/app.js

```diff
@@ -1,21 +1,34 @@
+/* eslint-disable no-console */
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
+const http = require('http');
+const fs = require('fs');
+const path = require('path');
 
-module.exports = sum;
+const PORT = process.env.port || 3000;
+
+const server = http.createServer((req, res) => {
+  const normalizedURL = new URL(req.url, `http://${req.headers.host}`);
+  const filePath = normalizedURL.pathname;
+  const fileName = path.basename(filePath) || 'index.html';
+  const dirName = path.dirname(filePath);
+
+  if (dirName !== '/file') {
```

Comment:
I agree that the task message maybe a little bit confusing, `Return only files from a folder public` but I think all subfolders in the public folder also should be accessible, that's how it usually works. With you current condition any subfolder will not work. 
![image](https://user-images.githubusercontent.com/45945732/233775997-c5b2bbe6-94ea-4631-a509-80b96cfff399.png)